inc/container.class.php

```diff
@@ -68,6 +68,44 @@ static function install(Migration $migration, $version) {
          $migration->migrationOneTable($table);
       }
 
+      // Fix containers names that were generated prior to Fields 1.9.2.
+      $glpi_version = preg_replace('/^((\d+\.?)+).*$/', '$1', GLPI_VERSION);
+      $bad_named_containers = $DB->request(
+         [
+            'FROM' => self::getTable(),
+            'WHERE' => [
+               // Regex will be escaped by PDO in GLPI 10+, but has to be escaped for GLPI < 10
+               'name' => ['REGEXP', version_compare($glpi_version, '10.0', '>=') ? '\d+' : '\\\d+'],
```

Comment:
Maybe better to use escaping method rather than hardcode it

Reply:
Done.